README.md

```diff
@@ -98,6 +98,29 @@ private void ExampleFunction()
 }
 ```
 
+**Access modifiers** are always writen with functions.
+```cs
+void ExampleFunction()
+{
+    Debug.Log("Not allowed");
+}
+
+private void ExampleFunction()
+{
+    Debug.Log("I'm a private function.");
+}
+
+protected void ExampleFunction()
+{
+    Debug.Log("I'm a protected function.");
+}
+
+public void ExampleFunction()
+{
+    Debug.Log("I'm a public function.");
```

Comment:
I would use summaries instead of `Debug.Log`

Reply:
That is an option, but to be consitant we use Debug.Log here.